wasmcc/wasmcc.go

```diff
@@ -22,8 +22,19 @@ import (
 	wasm_validation "github.com/perlin-network/life/wasm-validation"
 )
 
-const CHAINCODE_EXISTS = "{\"code\":101, \"reason\": \"chaincode exists with same name\"}"
-const UKNOWN_ERROR = "{\"code\":301, \"reason\": \"uknown error : %s\"}"
+//Exception messages for WASMCC
+const (
+	ChaincodeExists = "{\"code\":401, \"reason\": \"chaincode exists with same name\"}"
+	UnknownError    = "{\"code\":402, \"reason\": \"unknown error : %s\"}"
+	FnNotPresent    = "{\"code\":403, \"reason\": \"fn doesn't exists in installed wasm chaincode : %s\"}"
+)
```

Comment:
Why are these json formatted?

Reply:
these are being returned as wasmcc execution response. So if someone using sdk, would be easier for parsing.

Plus, reference from fabric-samples :smiley:

Reply:
Thanks for pointing out. Fixed it

Reply:
I see. This sets off a bit of alarm for me. Given the reason and the simplicity, we can allow it for now.

It'd probably be overengineering to insist on a struct and json marshalling. 

For interoperability reasons, having a defined struct in a separate package would be best for golang parsing of the messages, but would not help any other language.